<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberReg</class>
 <widget class="QDialog" name="MemberReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>431</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="firstLabel">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="firstInput"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lastInput">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="userInput">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="userLabel">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lastLabel">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="gradeDropDown">
      <property name="font">
       <font>
        <family>Roboto</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Freshman</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sophomore</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Junior</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Senior</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="passwordInput">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="passwordCheck">
      <property name="text">
       <string>Show Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
